### PR TITLE
Add missing (and update existing) banks in Croatia

### PR DESCRIPTION
**Pull request by mnalis**
as officially listed at: 
https://www.hnb.hr/temeljne-funkcije/supervizija/popis-kreditnih-institucija

### CODE REVIEW

File: data/brands/amenity/payment_centre.json [hunk 1]
```diff
@@ -71,6 +71,20 @@
...
+      "displayName": "Fina - Financijska agencija",
+      "locationSet": {"include": ["hr"]},
+      "tags": {
+        "brand": "Financijska agencija",
+        "brand:wikidata": "Q12631015",
+        "name": "Fina",
+        "amenity": "payment_centre",
+        "bureau_de_change": "yes",
+        "money_transfer": "western_union",
+        "office": "financial",
```
Comment (Cj-Malone) [inline data/brands/amenity/payment_centre.json:84]: I'm not fond of things getting multiple top level tags, in this case `amenity` and `office`. Are you sure this is correct tagging?

Comment (mnalis) [reply data/brands/amenity/payment_centre.json:84]: Yes, it seems most precise... While most citizens will use them for paying bills (thus `amenity=payment_centre`), they do offer more advanced financial services to SMEs too. 

But if you think it is problematic, I can remove `office=financial`?

Comment (Cj-Malone) [reply data/brands/amenity/payment_centre.json:84]: I would not have added `office=financial`, however it's not a hard rule, NSI does have entries with 2 categories, eg FedEx and Chuck e cheese.

I'll leave it for other people to provide opinions.

Comment (kjonosm) [reply data/brands/amenity/payment_centre.json:84]: I would also be in favour of removing `office=financial`. Besides, OSM wiki [claims](https://wiki.openstreetmap.org/wiki/Tag:office%3Dfinancial_advisor#See_also) `office=financial` should be used for offices not open to the general public.

Comment (mnalis) [reply data/brands/amenity/payment_centre.json]: Thanks for feedback, I've removed  `office=financial` now.

**Comment by Cj-Malone** [comment]
Thank you, it can be restored of changed at anytime if people have different opinions.

**Pull request merged**